src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Compatibility.Common.targets

```diff
@@ -0,0 +1,36 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
+<Project>
+  <UsingTask TaskName="Microsoft.DotNet.PackageValidation.ValidatePackage" AssemblyFile="$(DotNetPackageValidationAssembly)" />
+
+  <Target Name="RunPackageValidation"
+          AfterTargets="Pack"
+          Condition="'$(IsPackable)' == 'true' and '$(EnablePackageValidation)' == 'true'">
```

Comment:
Should the package set `EnablePackageValidation` to `true` on it's props file? It seems weird to reference the package and still needing to set the opt-in flag.

Reply:
Resolved offline